diff -p -up zinnia-0.05/perl/Makefile.PL.bindings zinnia-0.05/perl/Makefile.PL
--- zinnia-0.05/perl/Makefile.PL.bindings	2008-09-13 18:59:36.000000000 +0200
+++ zinnia-0.05/perl/Makefile.PL	2009-11-19 14:17:39.000000000 +0100
@@ -3,8 +3,8 @@ WriteMakefile(
     'NAME'		=> 'zinnia',
     'CC'                => 'c++',
     'LD'                => 'c++',
-    'INC'               => '',
-    'LIBS'              => '-lzinnia',
+    'INC'               => '-I../',
+    'LIBS'              => '-L../.libs -lzinnia',
 #    'VERSION'	        => '0.1',
     'OBJECT'            => 'zinnia_wrap.o' 
 );
